workers/api-server/src/http/wallet.rs

```diff
@@ -423,9 +423,11 @@ impl TypedHandler for CreateOrderHandler {
 
         let task = UpdateWalletTaskDescriptor::new_order(
```

Comment:
nit: could be nice to have `new_order` default to the global and add a separate `new_order_with_matching_pool`